edit.go

```diff
@@ -261,6 +285,17 @@ func (v *View) writeRune(x, y int, ch rune) error {
 		chr:     ch,
 	}
 
+	// magic operation in https://github.com/jesseduffield/gocui/pull/10
```

Comment:
I don't think we should reference to other issues

Reply:
deleted